service/src/app/client.py

```
@@ -89,29 +172,27 @@ def main():
 
 
 @callback(
-    Output(component_id="graph", component_property="figure"),
-    Output(component_id="table", component_property="rowData"),
+    Output(component_id="faculty_persons_graph", component_property="figure"),
     Input(component_id="submit", component_property="n_clicks"),
     State(component_id="input", component_property="value"),
 )
 def update_graph(clicks, input):
```

Ruff (0.15.12)

[error] Function argument `input` is shadowing a Python builtin

(A002)